packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -512,20 +580,23 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   }
 
   .mdc-textfield__label {
+    @include mdc-rtl-reflexive-position(left, 1px);
+
     top: $label-offset-y;
     bottom: initial;
-    left: $label-offset-x;
+    width: calc(100% - 38px);
```

Comment:
What's the significance of 38px? This isn't a remnant from the textarea icon, is it? (And if so, is there anything else here we missed?)

Reply:
The significance isn't because of the icon, but because the label needs to span most of the width of the textarea. It needs to span that so it can have an opaque background for when there is a long text entry where text would overflow the space, causing the label to overlay the text, and a scroll bar to appear.